scripts/new_project.kts

```diff
@@ -39,13 +40,31 @@ object NewProject {
     }
 
     private fun copyFiles(fromPath: String, toPath: String) {
-        val targetFolder = File("../$toPath")
-        val sourceFolder = File("../$fromPath")
+        val targetFolder = File(getRootPath() + toPath)
+        val sourceFolder = File(getRootPath() + fromPath)
         sourceFolder.copyRecursively(targetFolder, true) { file, exception ->
             showMessage(exception?.message ?: "Error copying files")
             return@copyRecursively OnErrorAction.TERMINATE
         }
     }
+
+    private fun cleanNewProjectFolder() {
+        executeCommand("sh " + getRootPath() + appNameWithoutSpace + File.separator + "gradlew -p " + getRootPath() + appNameWithoutSpace + " clean")
+        executeCommand("sh " + getRootPath() + appNameWithoutSpace + File.separator + "gradlew -p " + getRootPath() + appNameWithoutSpace + "/buildSrc clean")
+        File(getRootPath() + appNameWithoutSpace + File.separator + ".idea")?.deleteRecursively()
+        File(getRootPath() + appNameWithoutSpace + File.separator + ".gradle")?.deleteRecursively()
+        File(getRootPath() + appNameWithoutSpace + File.separator + "buildSrc" + File.separator + ".gradle")?.deleteRecursively()
+        File(getRootPath() + appNameWithoutSpace + File.separator + ".git")?.deleteRecursively()
+    }
+
+    private fun executeCommand(command: String) {
+        val process = Runtime.getRuntime().exec(command)
+        process.inputStream.reader().forEachLine { println(it) }
+    }
+
+    private fun getRootPath(): String {
+        return System.getProperty("user.dir").replace("scripts", "")
+    }
```

Comment:
I see there's a lot of redundancy on `getRootPath() + appNameWithoutSpace`

Should we bundle them to a top-level variable so we can re-use it anywhere?

Reply:
Updated again

Reply:
Sorry to keep on going on, but wouldn't something like this be a cleaner solution 🙇 
```
    private val rootPath: String
        get() = System.getProperty("user.dir").replace("scripts", "")

    private val projectPath: String
        get() = rootPath + appNameWithoutSpace
```

I'm open to other suggestions 🙏

Reply:
It's okay. I was wondering about the same thing too. Updated ✌️